app/e2e/client/helper.js

```diff
@@ -122,6 +124,19 @@ export async function readFileAsArrayBuffer(file) {
 	});
 }
 
+export function generateMnemonicPhrase(n) {
+	const result = new Array(n);
+	let len = words.length;
+	const taken = new Array(len);
+
+	while (n--) {
+		const x = Math.floor(Math.random() * len);
+		result[n] = words[x in taken ? taken[x] : x];
+		taken[x] = --len in taken ? taken[len] : len;
+	}
+	return result.join(' ');
```

Comment:
not good having spaces on the password, right? 😬 
```suggestion
	return result.join('-');
```
this char could be received as a param as well =)

Reply:
The space was actually discussed with the security team, we did it inspired by the metamask's secret recovery phrase... but I agree the separator can be a parameter